osmaxx/excerptexport/static/excerptexport/scripts/maps/excerpt_viewer.js

```diff
@@ -1,8 +1,22 @@
-window.ExcerptViewer = function(mapElementID, excerptApiUrl) {
+window.ExcerptViewer = function(mapElementID, excerptApiUrl, disableZoom) {
```

Comment:
I don't think that he knew about JavaScript ;-)

Translating to JavaScript it would give us:
```javascript
function(mapElementID, excerptApiUrl, options) {
    if (!options) options = {};
    if (!options.disableZoom) options.disableZoom = False;
...
}
``

I don't think this makes this in any way clearer.